lib/osf-components/addon/components/registries/schema-block-renderer/template.hbs

```diff
@@ -0,0 +1,13 @@
+<Registries::SchemaBlockRenderer::Mapper ...attributes as |renderers|>
+    {{#let (get renderers @schemaBlock.blockType) as |Renderer|}}
+        <Renderer
+            @changeset={{@changeset}}
```

Comment:
When you're updating this to use the strategy pattern, it might be good to pass all these args to the mapper/strategy component, and let it dole them out to the individual block renderers. That would decouple this `<Renderer />` invocation from the implementation of the block renderers -- they can be different (e.g. only one of them needs `@node`), and the strategy component gives them each exactly what they need.